app/controllers/api/v1/assignments_controller.rb

```diff
@@ -87,30 +87,29 @@ def remove_participant
   # the method is not working as of now because rails is not allowing the course id to be null
```

Comment:
The comments should be meaningful and document the function instead of warning that it's not working. Please keep the structure of comments similar for every method; you can refer to other controllers for this, for example user controller. Input and output comments are not needed.